aws.mk

```diff
@@ -27,18 +27,20 @@ create-eks: ## Create a new EKS cluster
 import-aws-vars:
 -include $(TF_EKS_STATE_DIR)/outputs.env
 
-# Currently only supports amd64
 .PHONY: build-push
 build-push: import-aws-vars ci-build ci-docker-build ## Build the operator image and push it to the GAR repository
 	aws ecr get-login-password --region $(AWS_REGION) | docker login --username AWS --password-stdin $(ECR_URL)
 	docker push $(IMG)
 
 .PHONY: integration-test-eks
+integration-test-eks: export SKIP_AWS_TESTS=false
```

Comment:
nit: any reason we couldn't pass these to the `$(MAKE) integration-test` below?

Reply:
Yeah it's setup this way to essentially override the "defaults" set in `Makefile`; I had trouble getting that to work when passing them directly to the make command.

Reply:
ok, no worries then.